.stylelintrc.json

```diff
@@ -1,6 +1,482 @@
 {
   "extends": "stylelint-config-sass-guidelines",
+  "plugins": [
+    "stylelint-declaration-strict-value"
+  ],
   "rules": {
+    "order/properties-alphabetical-order": null,
```

Comment:
Alphabetical order was disabled to use a type grouping instead, as discussed in the frontend chapter meetings.

Reply:
Finally! :clap: :clap: